src/external/OCSDK/IOCSDKLogData.ts

```diff
@@ -3,6 +3,7 @@ export default interface IOCSDKLogData {
     Event: string;
     Region: string;
     ElapsedTimeInMilliseconds: number;
+    HttpRequestResponseTime: number;
```

Comment:
arent this changes depending on OCSDK?, I think we should wait to get the version from OCSDK and then make the uptake along with this changes